DataFormats/L1TrackTrigger/interface/TTTrack.h

```diff
@@ -56,52 +70,59 @@ class TTTrack {
   }
 
   /// Track momentum
-  GlobalVector getMomentum(unsigned int nPar = 4) const;
-  void setMomentum(GlobalVector aMomentum, unsigned int nPar = 5);
+  GlobalVector momentum() const;
+  GlobalVector getMomentum(unsigned int npar = 4) const;
 
   /// Track curvature
-  double getRInv(unsigned int nPar = 4) const;
-  void setRInv(double aRInv, unsigned int nPar = 5);
+  double rInv() const;
+  double getRInv(unsigned int npar = 4) const;
```

Comment:
since `4` is used repeatedly as a default argument, it could be made into a named constexpr.

Reply:
fixed